### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by BenjaminGrabow**
##  What does this PR do
This Pull Request is for updating the FAQ page

##  Description of task to be completed

- update FAQ page

- delete all errors in the console

##  How should this manually be tested

- locally: go to the Netlify dev link and test the application

##  What are the relevant Trello board stories
[Trello Card](https://trello.com/c/nrV6Arh6/104-as-a-user-i-want-to-be-able-to-see-a-faq-page)

### CODE REVIEW

**Comment by BenjaminGrabow** [comment]
[As a user I want to be able to see a FAQ page](https://trello.com/c/nrV6Arh6/104-as-a-user-i-want-to-be-able-to-see-a-faq-page)

File: src/components/FAQ/GeneralFaq.js [hunk 1]
```diff
@@ -69,33 +59,10 @@ const General = props => {
...
-          {text3 && (
-            <AnsweredQuestionDiv>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-              Nulla id tristique magna, et gravida nibh. Phasellus
-              tempus sem a mi consectetur, ac ultricies mauris
-              condimentum. Duis nec metus ante. Morbi feugiat mauris
-              non lectus sodales sodales. Proin eu auctor elit,
-              facilisis tincidunt velit. Sed in lectus non urna varius
-              eleifend ac ut ligula. In eleifend at mi in molestie.
+              <li>Choose a Quality you want to improve</li>
```
Comment (domeccleston) [inline src/components/FAQ/GeneralFaq.js]: Choose a quality you want to improve

File: src/components/FAQ/PaymentFaq.js [hunk 2]
```diff
@@ -43,18 +38,31 @@ const Payment = props => {
               showImage1(display);
             }}
           >
-            <p>How do I update my profile?</p>
+            <p>How does our payment system works?</p>
```
Comment (domeccleston) [inline src/components/FAQ/PaymentFaq.js]: How does our payment system work?

File: src/components/FAQ/PaymentFaq.js [hunk 3]
```diff
@@ -43,18 +38,31 @@ const Payment = props => {
...
           {text1 && (
             <AnsweredQuestionDiv>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-              Nulla id tristique magna, et gravida nibh. Phasellus
-              tempus sem a mi consectetur, ac ultricies mauris
-              condimentum. Duis nec metus ante. Morbi feugiat mauris
-              non lectus sodales sodales. Proin eu auctor elit,
-              facilisis tincidunt velit. Sed in lectus non urna varius
-              eleifend ac ut ligula. In eleifend at mi in molestie.
+              <li>Devcoach will take care about the payments</li>
```
Comment (domeccleston) [inline src/components/FAQ/PaymentFaq.js]: Devcoach will take care of the payments

File: src/components/FAQ/PaymentFaq.js [hunk 4]
```diff
@@ -43,18 +38,31 @@ const Payment = props => {
...
+              </li>
+              <li>
+                We are the middleman between the students and the
+                coaches
+              </li>
+              <li>
+                Coaches only get paid if the interview happenend
+              </li>
+              <li>
+                If an appointment doesn't happened, the student will
```
Comment (domeccleston) [inline src/components/FAQ/PaymentFaq.js]: If an appointment doesn't happen,

File: src/components/FAQ/PaymentFaq.js [hunk 5]
```diff
@@ -43,18 +38,31 @@ const Payment = props => {
...
+                get the money from DevCoach back
+              </li>
+              <li>
+                Booth, the coach and the student are required to prove
```
Comment (domeccleston) [inline src/components/FAQ/PaymentFaq.js]: Both the coach and student are required

File: src/components/FAQ/PaymentFaq.js [hunk 6]
```diff
@@ -64,39 +72,14 @@ const Payment = props => {
...
-          {text3 && (
-            <AnsweredQuestionDiv>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-              Nulla id tristique magna, et gravida nibh. Phasellus
-              tempus sem a mi consectetur, ac ultricies mauris
-              condimentum. Duis nec metus ante. Morbi feugiat mauris
-              non lectus sodales sodales. Proin eu auctor elit,
-              facilisis tincidunt velit. Sed in lectus non urna varius
-              eleifend ac ut ligula. In eleifend at mi in molestie.
+              Currently we are providing:
```
Comment (domeccleston) [inline src/components/FAQ/PaymentFaq.js]: Currently we provide:

**Comment by BenjaminGrabow** [comment]
> Looking good! Thanks for getting this done, have left a few comments for suggested grammar tweaks

Thanks Dom, I made the changes with the new commit. We cannot have grammatical errors in production, if you find some more, please request new changes.

**Pull request merged**